assembly/src/parsers/mod.rs

```diff
@@ -160,10 +160,21 @@ fn parse_op_token(
         "cdropw" => stack_ops::parse_cdropw(span_ops, op),
 
         // ----- input / output operations --------------------------------------------------------
-        "push" => io_ops::parse_push(span_ops, op, num_proc_locals),
-        "pushw" => io_ops::parse_pushw(span_ops, op, num_proc_locals),
+        "push" => io_ops::parse_push(span_ops, op),
+        "sdepth" => io_ops::parse_env_ops(span_ops, op, num_proc_locals),
+        "locaddr" => io_ops::parse_env_ops(span_ops, op, num_proc_locals),
```

Comment:
As far as I can tell, there isn't much common logic between parsing `sdepth` and `locaddr`. Let's have 2 separate parse methods for them.